generators/csharp/codegen/src/asIs/RawClient.Template.cs

```diff
@@ -27,7 +26,16 @@ internal class RawClient(ClientOptions clientOptions)
     /// </summary>
     public readonly ClientOptions Options = clientOptions;
 
-    public async Task<ApiResponse> MakeRequestAsync(
+    [Obsolete("Use SendRequestAsync instead.")]
+    public Task<ApiResponse> MakeRequestAsync(
+        BaseApiRequest request,
+        CancellationToken cancellationToken = default
+    )
+    {
+        return SendRequestAsync(request, cancellationToken);
+    }
```

Comment:
The `RawClient` is internal, so we can also just remove this entirely. We can also do that in a follow-up if you prefer.